EIC-EA/EIC-community-D8
# EICNET-1170: Group about page: move the hardcoded text from the theme to the module

### Improvements

- Create new helper method in EICGroupsHelper to get the Group Flex plugin title and re-factor include files in the theme;
- Create class for group joining methods constants.

### Tests

- Create a group
- Go to the group about page and check if the labels and descriptions remain the same
